backend/src/Cli/Cli.fsproj

```diff
@@ -4,10 +4,9 @@
     <OutputType>Exe</OutputType>
     <TargetFramework>net8.0</TargetFramework>
     <LangVersion>8.0</LangVersion>
-    <!-- Publishing configuration -->
-    <IncludeNativeLibrariesForSelfExtract>true</IncludeNativeLibrariesForSelfExtract>
```

Comment:
Maybe it was a mistake to remove this -- I had done so while fighting with msbuild